e2e-tests/page-objects/custom-signin-page.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+'use strict';
+
+const util = require('./shared/util');
+
+function input(field) {
+  const inputWrap = `o-form-input-${field}`;
+  return $(`${util.se(inputWrap)} input`);
+}
+
+class CustomSignInPage {
+
+  constructor() {
+    this.usernameInput = input('username');
+    this.passwordInput = input('password');
+    this.submitButton = $('[data-type="save"]');
+    // TODO - Check if we can check on widget logo/text for page load
+    // Currently there's no different between this page and okta-signin-page
```

Comment:
A possible solution: check the current host of the page.  If you moved `http://localhost:8080` to a browser param, e.g. `browser.params.appRoot` then you could easily check that were's still on she same domain (rather than having been taken away to the hosted sign in page).

Reply:
Done